gradio/documentation.py

```diff
@@ -0,0 +1,146 @@
+import inspect
+
+classes_to_document = {}
+gl = {"mode": None}
```

Comment:
It's not clear what `gl` means, can we choose a clearer variable name?

Reply:
fixed